test/tools/libtesteth/ImportTest.cpp

```diff
@@ -289,11 +296,18 @@ json_spirit::mObject& ImportTest::makeAllFieldsHex(json_spirit::mObject& _o, boo
 			str = value.get_str();
 		else if (value.type() == json_spirit::array_type)
```

Comment:
thats what I am trying to make a point here. 
make allFields just convert the values. there should be no recursion.
it is just that there is a case when value could be an array of values